src/plugins/intel_cpu/src/nodes/embedding_bag_offsets.cpp

```diff
@@ -27,26 +29,38 @@ bool EmbeddingBagOffsetSum::isSupportedOperation(const std::shared_ptr<const ov:
     return true;
 }
 
-EmbeddingBagOffsetSum::EmbeddingBagOffsetSum(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr context)
+EmbeddingBagOffset::EmbeddingBagOffset(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr context)
     : Node(op, context, NgraphShapeInferFactory(op, EMPTY_PORT_MASK)),
-      EmbeddingBagSum(op, 3lu, 1lu, 4lu, 3lu) {
+      EmbeddingBag(op, 3lu, 1lu, 4lu, 3lu) {
     std::string errorMessage;
     if (!isSupportedOperation(op, errorMessage)) {
         OPENVINO_THROW_NOT_IMPLEMENTED(errorMessage);
     }
-
+    auto offsets_op = ov::as_type_ptr<ov::op::util::EmbeddingBagOffsetsBase>(op);
+    if (offsets_op) {
+        using OpReduction = ov::op::util::EmbeddingBagOffsetsBase::Reduction;
+        switch (offsets_op->get_reduction()) {
+        case OpReduction::SUM:
+            _reduction = Reduction::SUM;
+            break;
+        case OpReduction::MEAN:
+        default:
+            _reduction = Reduction::MEAN;
+            break;
+        }
+    }
```

Comment:
Do we need throw some exception for the else-branch?

Reply:
Added exception for cases where Reduction wouldn't be recognized